tests/system_tests/psyclone/test_psyclone_system_test.py

```diff
@@ -160,17 +161,17 @@ def test_run(self, config):
             config.build_output / 'algorithm/algorithm_mod_psy.f90',
 
             # Expect these prebuild files
-            # todo: the kernal hash differs between fpp and cpp, perhaps just use wildcards.
-            config.prebuild_folder / 'algorithm_mod.1602753696.an',  # x90 analysis result
-            config.prebuild_folder / 'my_kernel_mod.4187107526.an',  # kernel analysis results
-            config.prebuild_folder / 'algorithm_mod.5088673431.f90',  # prebuild
-            config.prebuild_folder / 'algorithm_mod_psy.5088673431.f90',  # prebuild
+            # The kernel hash differs between fpp and cpp, so just use wildcards.
+            config.prebuild_folder / 'algorithm_mod.*.an',  # x90 analysis result
+            config.prebuild_folder / 'my_kernel_mod.*.an',  # kernel analysis results
+            config.prebuild_folder / 'algorithm_mod.*.f90',  # prebuild
+            config.prebuild_folder / 'algorithm_mod_psy.*.f90',  # prebuild
         ]
 
-        assert all(not f.exists() for f in expect_files)
+        assert all((glob.glob(str(f)) == []) for f in expect_files)
         with config:
             self.steps(config)
-        assert all(f.exists() for f in expect_files)
+        assert all((glob.glob(str(f)) != []) for f in expect_files)
```

Comment:
See previous comment.

Reply:
Ditto, used Path's glob method, and a list (since otherwise the test will compare if an generator is != [] ... which is the case even if the iterator return nothing).